docs/recipes/Code-Splitting-Using-Holocron.md

```diff
@@ -1,7 +1,240 @@
+[holocron-readme]: https://github.com/americanexpress/holocron/tree/master/packages/holocron/README.md
+[holocron-module-route-readme]: https://github.com/americanexpress/holocron/tree/master/packages/holocron-module-route/README.md
+[one-app-bundler-readme]: https://github.com/americanexpress/one-app-cli/tree/master/packages/one-app-bundler/README.md
+[one-app-router-readme]: https://github.com/americanexpress/one-app-router/README.md
+[frank-lloyd-root]: ../../prod-sample/sample-modules/frank-lloyd-root/0.0.0/README.md
+[franks-burgers]: ../../prod-sample/sample-modules/franks-burgers/0.0.0/README.md
+
 [👈 Return to Overview](./Recipes.md)
 
 # Code Splitting Using Holocron
 
-Recipe is forthcoming.
+For One App, Holocron modules are a formal way of code-splitting for a modern,
+micro front-end in the One App ecosystem. Utilizing the module map as a system
+which promotes synergy and asynchrony across teams and collaborators on
+a single application.
+
```

Comment:
Let's stick to our formatting and have a nice Table of Contents:
Example:
```md
**Contents**
* [Shape](#shape)
* [Example](#example)
* [Field Annotations](#field-annotations)
  * [`key` Field](#key-field)
  * [`[moduleName]` Field](#moduleName-field)
  * [`browser` Field](#browser-field)
  * [`legacyBrowser` Field](#legacyBrowser-field)
  * [`node` Field](#node-field)
  * [`url` Field](#url-field)
  * [`integrity` Field](#integrity-field)
```

Reply:
For sure, will update

Reply:
updated in cb5e958